clang/lib/Serialization/ASTReader.cpp

```diff
@@ -833,32 +833,33 @@ bool SimpleASTReaderListener::ReadPreprocessorOptions(
 /// against the header search options in an existing preprocessor.
 ///
 /// \param Diags If non-null, produce diagnostics for any mismatches incurred.
-static bool checkHeaderSearchOptions(const HeaderSearchOptions &HSOpts,
+static bool checkHeaderSearchOptions(llvm::vfs::FileSystem &VFS,
```

Comment:
I realize you're not changing the _behaviour_ here, but now that we don't even pass in a `HeaderSearchOptions` we should probably rename this. Maybe `checkModuleCachePath`?